blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServer.scala

```diff
@@ -181,6 +185,7 @@ object BlazeBuilder extends BlazeBuilder(
   serviceExecutor = Strategy.DefaultExecutorService,
   idleTimeout = IdleTimeoutSupport.DefaultIdleTimeout,
   isNio2 = false,
+  enableWebSockets = false,
```

Comment:
Fair enough, I've changed it to enabled by default. I was trying to be conservative, but I guess we are still pre 1.0 so we might as well get it under some stress to press out the wrinkles.